src/commcare_cloud/ansible/roles/pgbouncer/tasks/main.yml

```
@@ -1,39 +1,55 @@
 ---
 
-- name: Install pgbouncer
-  apt: name=pgbouncer state=present
-  notify: Start pgbouncer
+- name: pgbouncer install
+  package: name=pgbouncer state=present
   tags:
     - pgbouncer
 
-- name: pgbouncer configuration
-  template: src=pgbouncer.ini.j2 dest="{{ pgbouncer_ini }}"
-  notify: Reload pgbouncer
+- name: pgbouncer gather package facts
+  package_facts:
+    manager: auto
   tags:
     - pgbouncer
-    - configure
 
-- name: pgbouncer users
-  template: src=pgbouncer.users.j2 dest="{{ pgbouncer_users }}"
-  notify: Reload pgbouncer
+- name: pgbouncer quit if version is too old
+  fail:
+    msg: "pgbouncer version currently installed is too old"  
```

Add more context to error message: 
```suggestion
    msg: "pgbouncer version currently installed is too old: {{ ansible_facts.packages['pgbouncer'][0].version }} < {{ pgbouncer_min_version }}"  
```

---

Added a second line.